charts/tailscale-subnet-router/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: tailscale-subnet-router
 description: Deploy a Tailscale subnet router on Kubernetes
 type: application
-version: 1.1.1
-appVersion: "v1.30.0"
+version: 1.1.2
```

Comment:
This was current when you issued the PR, but I've been really slow to review. Could I bug you to rebase and bump just the `version`?